### PR TITLE
Xcas.Xcas version 1.6.0

### PR DESCRIPTION
**Pull request by superusercode**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56668)

### CODE REVIEW

File: manifests/x/Xcas/Xcas/1.6.0/Xcas.Xcas.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Xcas.Xcas
+PackageVersion: 1.6.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: http://www-fourier.ujf-grenoble.fr/~parisse/giac/xcaswininst.exe
+  InstallerSha256: 43F250FD412F21511AB156633FAB26C4D6A52E542145BD4B80C18E3D033F5FAF
```
Comment (ItzLevvie) [inline manifests/x/Xcas/Xcas/1.6.0/Xcas.Xcas.installer.yaml:10]:
```suggestion
  InstallerSha256: 43F250FD412F21511AB156633FAB26C4D6A52E542145BD4B80C18E3D033F5FAF
  ProductCode: Xcas for Windows
```

File: manifests/x/Xcas/Xcas/1.6.0/Xcas.Xcas.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Xcas.Xcas
+PackageVersion: 1.6.0
+PackageLocale: en-US
+Publisher: Xcas
+PackageName: Giac/Xcas
+License: GNU GPLv2
```
Comment (ItzLevvie) [inline manifests/x/Xcas/Xcas/1.6.0/Xcas.Xcas.locale.en-US.yaml:9]:
```suggestion
License: GPL-2.0
```

File: manifests/x/Xcas/Xcas/1.6.0/Xcas.Xcas.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Xcas.Xcas
+PackageVersion: 1.6.0
+PackageLocale: en-US
+Publisher: Xcas
+PackageName: Giac/Xcas
```
Comment (ItzLevvie) [inline manifests/x/Xcas/Xcas/1.6.0/Xcas.Xcas.locale.en-US.yaml:8]:
```suggestion
PackageName: Xcas for Windows
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by superusercode** [comment]
Is the bot geoblocked from the URL? Not sure why this works locally for me but not for the bot.

**Comment by ItzLevvie** [comment]
Wait for an engineer to take a look.

**Pull request merged**